projects/cdk/schematics/ng-update/tests/schematic-migrate-sliders.spec.ts

```diff
@@ -129,28 +131,33 @@ const FILE_HTML_CHILD_TEMPLATE_BEFORE = `
     [maxLabel]="Maxxxxx"
     minLabel="min!!!"
     [formControl]="control"
+    [segmentsPluralize]="someVariable"
 >
     Select volume range
 </tui-input-range>
 
 <tui-slider [min]="100" [max]="1000" size="m" [segments]="5" [quantum]="0.01"></tui-slider>
 `;
 
-const FILE_HTML_CHILD_TEMPLATE_AFTER =
-    '\n' +
-    '<tui-input-slider\n' +
-    '    tuiHintContent="Select the answer to see how the right custom content changes"\n' +
-    '    [valueContent]="tuiMigrationMinMaxLabel"\n' +
-    '    \n' +
-    '    class="control"\n' +
-    '    [min]="0"\n' +
-    '    [max]="10"\n' +
-    "    [tuiTextfieldCustomContent]=\"userAnswer === 4 ? 'right' : 'wrong'\"\n" +
-    '    [(ngModel)]="userAnswer"\n' +
-    '>\n' +
-    '    2+2=?\n' +
-    '</tui-input-slider>\n' +
-    `
+const FILE_HTML_CHILD_TEMPLATE_AFTER = `
+<!-- TODO: [pluralize] => Use [postfix] instead. See https://taiga-ui.dev/components/input-slider/API?postfix=apples -->
+<!-- TODO: See examples how create labels for ticks without this property (outside the component): https://taiga-ui.dev/components/input-slider#slider-segments -->
+<tui-input-slider
+    tuiHintContent="Select the answer to see how the right custom content changes"
+    [valueContent]="tuiMigrationMinMaxLabel"
+    ${''}
+    class="control"
+    [min]="0"
+    [max]="10"
+    [tuiTextfieldCustomContent]="userAnswer === 4 ? 'right' : 'wrong'"
+    [pluralize]="pluralForms"
+    [segmentsPluralize]="pluralForms"
+    [(ngModel)]="userAnswer"
+>
+    2+2=?
+</tui-input-slider>
+
+<!-- TODO: See examples how create labels for ticks without this property (outside the component): https://taiga-ui.dev/components/input-range#segments -->
```

Comment:
How about we add `TODO: Taiga UI migration` to all such TODOs and if we did add anything we write at the end:
`Some migrations have to be done manually, search your codebase for added "TODO: Taiga UI migration" comments`

Reply:
I added `(Taiga UI migration)` to all TODO comments ✅

About this
> and if we did add anything we write at the end:
Some migrations have to be done manually, search your codebase for added "TODO: Taiga UI migration" comments

I will add this line in next PR